--- btrfsprogs.mk.orig	2022-07-30 20:02:30.685088000 +0200
+++ btrfsprogs.mk	2022-07-30 18:54:16.286833000 +0200
@@ -1,5 +1,5 @@
 btrfsprogs-configure: lzo zstd zlib
-	cd btrfsprogs && ./autogen.sh
+	cd btrfsprogs && autoreconf -fi
 	cd btrfsprogs && ./configure --host=$(ARCH)-linux \
 		CFLAGS="$(COPTS) $(MIPS16_OPT) -ffunction-sections -fdata-sections -Wl,--gc-sections -I$(TOP)/btrfsprogs -I$(TOP)/$(ARCH)-uclibc/install/util-linux/usr/include -I$(TOP)/e2fsprogs/lib -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -I$(TOP)/zlib -I$(TOP)/lzo/include -DNEED_PRINTF -I$(TOP)/zstd/lib" \
 		LDFLAGS="-L$(TOP)/zlib -L$(TOP)/$(ARCH)-uclibc/install/util-linux/usr/lib  -L$(TOP)/e2fsprogs/lib -L$(TOP)/lzo/src/.libs -lz -ffunction-sections -fdata-sections -Wl,--gc-sections" \
@@ -9,6 +9,7 @@
 		--disable-python \
 		--disable-backtrace \
 		--disable-documentation \
+		--disable-convert \
 		root_prefix=$(INSTALLDIR)/btrfsprogs \
 		ZLIB_CFLAGS=" " \
 		ZLIB_LIBS="-lz" \
