--- chillispot.mk.orig	2022-08-07 15:16:38.605451000 +0200
+++ chillispot.mk	2022-07-26 09:41:58.940258000 +0200
@@ -32,7 +32,7 @@
 
 chillispot-configure:
 	cd $(CHILLIDIR) && ./bootstrap
-	cd $(CHILLIDIR) &&  rm -rf config.{cache,status} && ./configure $(CHILLIEXTRAFLAGS) --host=$(ARCH)-linux-elf  \
+	cd $(CHILLIDIR) &&  rm -rf config.{cache,status} && ./configure $(CHILLIEXTRAFLAGS) --host=$(ARCH)-linux-gnu  \
 	    CFLAGS="$(COPTS) $(LTO) $(MIPS16_OPT) $(CHILLIEXTRA_CFLAGS) -fcommon -DHAVE_MALLOC=1 -Drpl_malloc=malloc -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	    LDFLAGS="$(LDLTO) -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	    AR_FLAGS="cru $(LTOPLUGIN)" \
