--- /home/stephan/dd-wrt/src/router/rules/chillispot.mk	2021-03-31 23:13:07.940000000 +0200
+++ chillispot.mk	2021-03-05 19:27:20.773260400 +0100
@@ -32,7 +32,7 @@
 
 chillispot-configure:
 	cd $(CHILLIDIR) && ./bootstrap
-	cd $(CHILLIDIR) &&  rm -rf config.{cache,status} && ./configure $(CHILLIEXTRAFLAGS) --host=$(ARCH)-linux-elf  \
+	cd $(CHILLIDIR) &&  rm -rf config.{cache,status} && ./configure $(CHILLIEXTRAFLAGS) --host=$(ARCH)-linux-gnu  \
 	    CFLAGS="$(COPTS) $(LTO) $(MIPS16_OPT) $(CHILLIEXTRA_CFLAGS) -fcommon -DHAVE_MALLOC=1 -Drpl_malloc=malloc -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	    LDFLAGS="$(LDLTO) -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	    AR_FLAGS="cru $(LTOPLUGIN)" \
