--- common.mk.orig	2022-08-07 15:16:19.415012000 +0200
+++ common.mk	2022-08-06 20:17:41.543062000 +0200
@@ -1,7 +1,9 @@
-export LTO := -flto -fwhole-program -flto-partition=none
-export LTOMIN := -flto
-export LDLTO := -flto=$(shell getconf _NPROCESSORS_ONLN) -fuse-linker-plugin
-export LTOPLUGIN := --plugin=$(shell $(CROSS_COMPILE)gcc --print-file-name=liblto_plugin.so)
+#export LTO := -flto -fwhole-program -flto-partition=none
+#export LTOMIN := -flto
+#export LDLTO := -flto=$(shell getconf _NPROCESSORS_ONLN) -fuse-linker-plugin
+#export LTOPLUGIN := --plugin=$(shell $(CROSS_COMPILE)gcc --print-file-name=liblto_plugin.so)
+#export LTOPLUGIN := --plugin=/home/stephan/dd-wrt_toolchains/toolchain-mipsel_74kc_gcc-10.1.0_musl/lib/gcc/mipsel-openwrt-linux-musl/10.1.0/liblto_plugin.so.0.0.0
+
 
 
 ifeq ($(ARCH),arm)
@@ -223,10 +225,10 @@
 	-cp /tmp/$(ARCHITECTURE)/mklibs-out/* $(TARGETDIR)/lib
 endif
 endif
-	../../tools/optimize_lib.sh libutils/ libutils.so libutils.a libutils_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libutils.so  $(TOP)/libutils/libutils_ld
-	../../tools/optimize_lib.sh libutils/ libwireless.so libwireless.a libwireless_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libwireless.so $(TOP)/libutils/libwireless_ld
-	../../tools/optimize_lib.sh libutils/ libshutils.so libshutils.a libshutils_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libshutils.so $(TOP)/libutils/libshutils_ld
-	../../tools/optimize_lib.sh nvram/ libnvram.so libnvram.a libnvram_min.so $(TARGETDIR) $(TARGETDIR)/lib/libnvram.so $(TOP)/nvram/libnvram_ld
+	../../tools/optimize_lib.sh libutils libutils.so libutils.a libutils_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libutils.so  $(TOP)/libutils/libutils_ld
+	../../tools/optimize_lib.sh libutils libwireless.so libwireless.a libwireless_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libwireless.so $(TOP)/libutils/libwireless_ld
+	../../tools/optimize_lib.sh libutils libshutils.so libshutils.a libshutils_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libshutils.so $(TOP)/libutils/libshutils_ld
+	../../tools/optimize_lib.sh nvram libnvram.so libnvram.a libnvram_min.so $(TARGETDIR) $(TARGETDIR)/lib/libnvram.so $(TOP)/nvram/libnvram_ld
 #	rm -f $(TARGETDIR)/lib/*.a
 #	rm -f $(TARGETDIR)/lib/*.map
 #	cp lib.$(ARCH)/libresolv.so.0 $(TARGETDIR)/lib
