--- common.mk.orig	2021-04-03 14:06:42.260000000 +0200
+++ common.mk	2021-04-03 16:24:04.770000000 +0200
@@ -1,7 +1,7 @@
-export LTO := -flto -fwhole-program
-export LTOMIN := -flto
-export LDLTO := -flto=jobserver -fuse-linker-plugin
-export LTOPLUGIN := --plugin=$(shell $(CROSS_COMPILE)gcc --print-file-name=liblto_plugin.so)
+export LTO := 
+export LTOMIN := 
+export LDLTO := 
+export LTOPLUGIN := 
 
 
 ifeq ($(ARCH),arm)
@@ -208,17 +208,17 @@
 		`cat /tmp/$(ARCHITECTURE)/mklibs-progs` 2>&1
 	cp /tmp/$(ARCHITECTURE)/mklibs-out/* $(TARGETDIR)/lib
 else
-	-mklibs.py -D \
-		-d /tmp/$(ARCHITECTURE)/mklibs-out \
-		--sysroot $(TARGETDIR) \
-		-L /lib \
-		-L /usr/lib \
-		-L /usr/lib/plexmediaserver/lib \
-		--ldlib /lib/$(MUSL_LD) \
-		--target $(ARCH)-linux-uclibc \
-		`cat /tmp/$(ARCHITECTURE)/mklibs-progs` 2>&1
+#	-mklibs.py -D \
+#		-d /tmp/$(ARCHITECTURE)/mklibs-out \
+#		--sysroot $(TARGETDIR) \
+#		-L /lib \
+#		-L /usr/lib \
+#		-L /usr/lib/plexmediaserver/lib \
+#		--ldlib /lib/$(MUSL_LD) \
+#		--target $(ARCH)-linux-uclibc \
+#		`cat /tmp/$(ARCHITECTURE)/mklibs-progs` 2>&1
 
-	-cp /tmp/$(ARCHITECTURE)/mklibs-out/* $(TARGETDIR)/lib
+#	-cp /tmp/$(ARCHITECTURE)/mklibs-out/* $(TARGETDIR)/lib
 endif
 endif
 	../../tools/optimize_lib.sh libutils/ libutils.so libutils.a libutils_min.so $(TARGETDIR) $(TARGETDIR)/usr/lib/libutils.so  $(TOP)/libutils/libutils_ld
