--- configure_python.ac.orig	2022-08-07 15:16:37.995437000 +0200
+++ configure_python.ac	2022-07-31 14:06:57.374145000 +0200
@@ -71,9 +71,9 @@
 	    fi
             interp=
 	done
-        if test x$interp = x; then
-	    AC_MSG_ERROR([python$PACKAGE_VERSION interpreter not found])
-	fi
+ #       if test x$interp = x; then
+#	    AC_MSG_ERROR([python$PACKAGE_VERSION interpreter not found])
+#	fi
         AC_MSG_RESULT($interp)
 	PYTHON_FOR_BUILD='_PYTHON_PROJECT_BASE=$(abs_builddir) _PYTHON_HOST_PLATFORM=$(_PYTHON_HOST_PLATFORM) PYTHONPATH=$(shell test -f pybuilddir.txt && echo $(abs_builddir)/`cat pybuilddir.txt`:)$(srcdir)/Lib _PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_$(ABIFLAGS)_$(MACHDEP)_$(MULTIARCH) '$interp
     fi
