--- daq.mk.orig	2022-08-07 15:16:38.605451000 +0200
+++ daq.mk	2022-08-05 20:22:18.279638000 +0200
@@ -1,6 +1,6 @@
 ifeq ($(CONFIG_IPV6),y)
-daq-configure: libpcap libdnet-configure libnetfilter_queue-configure libdnet libnetfilter_queue iptables-new
-	cd daq && autoconf
+daq-configure:
+	cd daq && autoreconf -fi
 	export ac_cv_header_linux_netfilter_h=yes ; \
 	export ac_cv_lib_pcap_pcap_lib_version=yes ; \
 	export daq_cv_libpcap_version_1x=yes ; \
@@ -21,10 +21,10 @@
 		CFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC -DNEED_PRINTF -D_GNU_SOURCE -Drpl_malloc=malloc -I$(TOP)/iptables-new/include -I$(TOP)/iptables-new/include/libipq -I$(TOP)/libnetfilter_queue/include -I$(TOP)/libnfnetlink/include" \
 		CPPFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC -DNEED_PRINTF -D_GNU_SOURCE  -Drpl_malloc=malloc -I$(TOP)/iptables-new/include -I$(TOP)/iptables-new/include/libipq -I$(TOP)/libnetfilter_queue/include -I$(TOP)/libnfnetlink/include" \
 		LDFLAGS="-L$(TOP)/iptables-new/libipq/.libs -L$(TOP)/libnetfilter_queue/src/.libs -L$(TOP)/libnfnetlink/src/.libs -fPIC"
-	$(MAKE) -C daq all install CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
+
 else
-daq-configure: libpcap libdnet-configure libnetfilter_queue-configure libdnet libnetfilter_queue iptables
-	cd daq && autoconf
+daq-configure:
+	cd daq && autoreconf -fi
 	export ac_cv_header_linux_netfilter_h=yes ; \
 	export ac_cv_lib_pcap_pcap_lib_version=yes ; \
 	export daq_cv_libpcap_version_1x=yes ; \
@@ -35,7 +35,6 @@
 		--enable-nfq-module \
 		--enable-ipq-module \
 		--enable-pcap-module \
-		--enable-static \
 		--prefix=$(TOP)/daq/install \
 		--libdir=$(TOP)/daq/install/lib \
 		--with-libpcap-includes="$(TOP)/libpcap" \
@@ -45,22 +44,25 @@
 		CFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC -DNEED_PRINTF -D_GNU_SOURCE -Drpl_malloc=malloc -I$(TOP)/iptables/include -I$(TOP)/iptables/include/libipq -I$(TOP)/libnetfilter_queue/include -I$(TOP)/libnfnetlink/include" \
 		CPPFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC -DNEED_PRINTF -D_GNU_SOURCE  -Drpl_malloc=malloc -I$(TOP)/iptables/include -I$(TOP)/iptables/include/libipq -I$(TOP)/libnetfilter_queue/include -I$(TOP)/libnfnetlink/include" \
 		LDFLAGS="-L$(TOP)/iptables/libipq -L$(TOP)/libnetfilter_queue/src/.libs -L$(TOP)/libnfnetlink/src/.libs -fPIC"
-	$(MAKE) -C daq all install CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
+
 endif
 daq: libpcap libdnet iptables
 	-mkdir daq/install
 	-rm -rf daq/install/*
-	$(MAKE) -C daq all install CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
+	$(MAKE) -C daq all CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
+	cd daq/os-daq-modules && chmod 755 daq-modules-config
+#	$(MAKE) -C daq install CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
 
 daq-clean:
 	-rm -rf daq/install/*
 	$(MAKE) -C daq clean  CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
 
 daq-install:
-	-install -D daq/install/lib/libsfbpf.so.0 $(INSTALLDIR)/daq/usr/lib/libsfbpf.so.0
-	-install -D daq/install/lib/daq/daq_ipq.so $(INSTALLDIR)/daq/usr/lib/daq/daq_ipq.so
-	-install -D daq/install/lib/daq/daq_nfq.so $(INSTALLDIR)/daq/usr/lib/daq/daq_nfq.so
-	-install -D daq/install/lib/daq/daq_pcap.so $(INSTALLDIR)/daq/usr/lib/daq/daq_pcap.so
+#	$(MAKE) -C daq install CFLAGS="$(COPTS) -fPIC -DNEED_PRINTF -Drpl_malloc=malloc"
+	-install -D daq/sfbpf/.libs/libsfbpf.so.0 $(INSTALLDIR)/daq/usr/lib/libsfbpf.so.0
+	-install -D daq/os-daq-modules/.libs/daq_ipq.so $(INSTALLDIR)/daq/usr/lib/daq/daq_ipq.so
+	-install -D daq/os-daq-modules/.libs/daq_nfq.so $(INSTALLDIR)/daq/usr/lib/daq/daq_nfq.so
+	-install -D daq/os-daq-modules/.libs/daq_pcap.so $(INSTALLDIR)/daq/usr/lib/daq/daq_pcap.so
 	-install -D daq/install/lib64/libsfbpf.so.0 $(INSTALLDIR)/daq/usr/lib/libsfbpf.so.0
 	-install -D daq/install/lib64/daq/daq_ipq.so $(INSTALLDIR)/daq/usr/lib/daq/daq_ipq.so
 	-install -D daq/install/lib64/daq/daq_nfq.so $(INSTALLDIR)/daq/usr/lib/daq/daq_nfq.so
