--- /home/stephan/dd-wrt/src/router/rules/glib.mk	2021-03-31 23:13:07.940000000 +0200
+++ glib.mk	2021-03-19 18:32:45.190000000 +0100
@@ -9,9 +9,8 @@
 endif
 
 
-glib20-configure: libffi zlib util-linux
-	make -C util-linux
-	make -C util-linux install DESTDIR=$(INSTALLDIR)/util-linux
+#glib20-configure: libffi zlib util-linux
+glib20-configure:
 	mkdir -p $(INSTALLDIR)/util-linux/usr/lib
 	-cp -urv $(INSTALLDIR)/util-linux/usr/tmp/* $(INSTALLDIR)/util-linux/usr/lib
 	rm -rf $(INSTALLDIR)/util-linux/usr/tmp 
@@ -25,49 +24,8 @@
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libblkid.la
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.so*
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.la
-
-	cd glib20/gettext && ./autogen.sh
-	cd glib20/gettext && ./configure --enable-static --disable-shared --host=$(ARCH)-linux  LDFLAGS="$(COPTS) $(LTO) -std=gnu89 $(MIPS16_OPT) -D_GNU_SOURCE -fPIC -Drpl_malloc=malloc " CFLAGS="$(COPTS)  $(MIPS16_OPT)  -D_GNU_SOURCE -fPIC -Drpl_malloc=malloc" CXXFLAGS="$(COPTS)  $(MIPS16_OPT) -D_GNU_SOURCE -fPIC -Drpl_malloc=malloc"
-	$(MAKE) -C glib20/gettext clean all
-
-	-cd glib20/libglib && ./autogen.sh
-	cd glib20/libglib && ./configure --enable-shared --enable-static --disable-fam --with-libiconv=gnu --disable-libelf --with-pcre=internal --disable-libmount --enable-debug=no --disable-selinux --disable-man --host=$(ARCH)-linux  CFLAGS="$(COPTS) $(LTO) -std=gnu89  -DNVALGRIND=1 $(MIPS16_OPT) -D_GNU_SOURCE=1  -I$(TOP)/zlib -fPIC -Drpl_malloc=malloc -I$(TOP)/glib20/gettext -I$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/include  -L$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/.libs -lffi -L$(TOP)/glib20/gettext/.libs -L$(TOP)/glib20/libglib/gmodule/.libs   -L$(TOP)/zlib -L$(TOP)/$(ARCH)-uclibc/install/util-linux/usr/lib -pthread -lpthread -lz" --disable-modular-tests \
-	LIBFFI_CFLAGS="-I$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/include" \
-	LIBFFI_LIBS="-L$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/.libs -lffi" \
-	ZLIB_CFLAGS="-I$(TOP)/zlib" \
-	ZLIB_LIBS="-L$(TOP)/zlib -lz" \
-	AR_FLAGS="cru $(LTOPLUGIN)" \
-	RANLIB="$(ARCH)-linux-ranlib $(LTOPLUGIN)" \
-	glib_cv_stack_grows=no glib_cv_uscore=no ac_cv_func_mmap_fixed_mapped=yes ac_cv_func_posix_getpwuid_r=yes ac_cv_func_posix_getgrgid_r=yes
-
-	$(MAKE) -C glib20/libglib clean all
-
-glib20: libffi zlib util-linux util-linux-install
-	make -C util-linux install DESTDIR=$(INSTALLDIR)/util-linux
-	mkdir -p $(INSTALLDIR)/util-linux/usr/lib
-	-cp -urv $(INSTALLDIR)/util-linux/usr/tmp/* $(INSTALLDIR)/util-linux/usr/lib
-	rm -rf $(INSTALLDIR)/util-linux/usr/tmp 
-	rm -rf $(INSTALLDIR)/util-linux/usr/sbin
-	rm -rf $(INSTALLDIR)/util-linux/usr/bin
-	rm -rf $(INSTALLDIR)/util-linux/bin
-	rm -rf $(INSTALLDIR)/util-linux/sbin
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libuuid.a
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libuuid.la
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libblkid.a
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libblkid.la
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.so*
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.la
-	$(MAKE) -C glib20/gettext all
-	$(MAKE) -C glib20/libglib all
-	rm -rf $(INSTALLDIR)/util-linux/usr/sbin
-	rm -rf $(INSTALLDIR)/util-linux/usr/bin
-	rm -rf $(INSTALLDIR)/util-linux/bin
-	rm -rf $(INSTALLDIR)/util-linux/sbin
-	rm -rf $(INSTALLDIR)/util-linux/usr/share
-	rm -rf $(INSTALLDIR)/util-linux/usr/include
 	rm -rf $(INSTALLDIR)/util-linux/usr/lib/pkgconfig
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.a
-	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.so*
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libmount.la
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libfdisk*
 	rm -f $(INSTALLDIR)/util-linux/usr/lib/libsmartcols*
@@ -90,8 +48,27 @@
 endif
 	rm -f $(INSTALLDIR)/util-linux/lib/libfdisk.so*
 	rm -f $(INSTALLDIR)/util-linux/lib/libsmartcols.so*
+	
+	cd glib20/gettext && ./autogen.sh
+	cd glib20/gettext && ./configure --enable-static --disable-shared --host=$(ARCH)-linux  LDFLAGS="$(COPTS) $(LTO) -std=gnu89 $(MIPS16_OPT) -D_GNU_SOURCE -fPIC -Drpl_malloc=malloc " CFLAGS="$(COPTS)  $(MIPS16_OPT)  -D_GNU_SOURCE -fPIC -Drpl_malloc=malloc" CXXFLAGS="$(COPTS)  $(MIPS16_OPT) -D_GNU_SOURCE -fPIC -Drpl_malloc=malloc"
+#	$(MAKE) -C glib20/gettext clean all
 
+	-cd glib20/libglib && ./autogen.sh
+	cd glib20/libglib && ./configure --enable-shared --enable-static --disable-fam --with-libiconv=gnu --disable-libelf --with-pcre=internal --disable-libmount --enable-debug=no --disable-selinux --disable-man --host=$(ARCH)-linux  CFLAGS="$(COPTS) $(LTO) -std=gnu89  -DNVALGRIND=1 $(MIPS16_OPT) -D_GNU_SOURCE=1  -I$(TOP)/zlib -fPIC -Drpl_malloc=malloc -I$(TOP)/glib20/gettext -I$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/include  -L$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/.libs -lffi -L$(TOP)/glib20/gettext/.libs -L$(TOP)/glib20/libglib/gmodule/.libs   -L$(TOP)/zlib -L$(TOP)/$(ARCH)-uclibc/install/util-linux/usr/lib -pthread -lpthread -lz" --disable-modular-tests \
+	LIBFFI_CFLAGS="-I$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/include" \
+	LIBFFI_LIBS="-L$(TOP)/libffi/$(ARCH)-$(SUBARCH)-linux-gnu/.libs -lffi" \
+	ZLIB_CFLAGS="-I$(TOP)/zlib" \
+	ZLIB_LIBS="-L$(TOP)/zlib -lz" \
+	AR_FLAGS="cru" \
+	RANLIB="$(ARCH)-linux-ranlib $(LTOPLUGIN)" \
+	glib_cv_stack_grows=no glib_cv_uscore=no ac_cv_func_mmap_fixed_mapped=yes ac_cv_func_posix_getpwuid_r=yes ac_cv_func_posix_getgrgid_r=yes
+
+#	$(MAKE) -C glib20/libglib clean all
 
+#glib20: libffi zlib util-linux util-linux-install
+glib20: 
+	$(MAKE) -C glib20/gettext all
+	$(MAKE) -C glib20/libglib all
 
 glib20-clean:
 	$(MAKE) -C glib20/gettext clean
