--- /home/stephan/dd-wrt/src/router/rules/igmp-proxy.mk	2021-03-31 23:13:07.940000000 +0200
+++ igmp-proxy.mk	2021-03-05 19:27:20.773260400 +0100
@@ -1,6 +1,6 @@
 igmp-proxy-configure:
 	cd igmp-proxy && ./autogen.sh
-	cd igmp-proxy && ./configure --host=$(ARCH)-linux-elf CFLAGS="$(COPTS) $(MIPS16_OPT) $(LTO) -I$(TOP)/kernel_headers/$(KERNELRELEASE)/include -ffunction-sections -fdata-sections -Wl,--gc-sections -std=gnu99"
+	cd igmp-proxy && ./configure --host=$(ARCH)-linux-gnu CFLAGS="$(COPTS) $(MIPS16_OPT) $(LTO) -I$(TOP)/kernel_headers/$(KERNELRELEASE)/include -ffunction-sections -fdata-sections -Wl,--gc-sections -std=gnu99"
 
 igmp-proxy-clean:
 	make -C igmp-proxy clean
