--- /home/stephan/dd-wrt/src/router/rules/jansson.mk	2021-03-31 23:13:07.940000000 +0200
+++ jansson.mk	2021-03-05 19:27:20.773260400 +0100
@@ -2,7 +2,7 @@
 	cd jansson && autoreconf --force --install
 	cd jansson && ./configure --prefix=/usr --libdir=/usr/lib --host=$(ARCH)-linux CC="$(CC)" CFLAGS="$(COPTS) $(MIPS16_OPT) -ffunction-sections -fdata-sections -Wl,--gc-sections -fPIC" LDFLAGS="-lm"
 
-jansson:
+jansson: jansson-configure
 	$(MAKE) -C jansson
 
 jansson-clean:
