--- /home/stephan/dd-wrt/src/router/rules/json-c.mk	2021-03-31 23:13:07.940000000 +0200
+++ json-c.mk	2021-03-05 19:27:20.773260400 +0100
@@ -7,9 +7,6 @@
 		--libdir=$(TOP)/_staging/usr/lib \
 		--includedir=$(TOP)/_staging/usr/include \
 		CFLAGS="-fPIC -DNEED_PRINTF $(COPTS) $(MIPS16_OPT)" LDFLAGS="-lm"
-	make -C json-c
-	-mkdir -p $(TOP)/_staging
-	make -C json-c install
 
 json-c:
 	make -C json-c
