--- /home/stephan/dd-wrt/src/router/rules/libevent.mk	2021-03-31 23:13:07.940000000 +0200
+++ libevent.mk	2021-03-05 19:27:20.773260400 +0100
@@ -18,12 +18,7 @@
 libevent-clean:
 	$(MAKE) -C libevent clean
 
-libevent-configure: openssl
-	cd libevent && ./configure  --disable-debug-mode --prefix=/usr ac_cv_host=$(ARCH)-uclibc-linux --libdir=/usr/lib --target=$(ARCH)-linux --host=$(ARCH) CC="ccache $(ARCH)-linux-uclibc-gcc" \
+libevent-configure: 
+	cd libevent && autoreconf -fi && ./configure  --disable-debug-mode --prefix=/usr ac_cv_host=$(ARCH)-uclibc-linux --libdir=/usr/lib --target=$(ARCH)-linux --host=$(ARCH) CC="ccache $(ARCH)-linux-uclibc-gcc" \
 	CFLAGS="$(COPTS) $(MIPS16_OPT) -I$(TOP)/openssl/include -ffunction-sections -fdata-sections -Wl,--gc-sections" \
-	LDFLAGS="$(COPTS) $(MIPS16_OPT) -L$(TOP)/openssl" 
-	CC="ccache $(ARCH)-linux-uclibc-gcc" \
-	CFLAGS="$(COPTS) $(MIPS16_OPT) -I$(TOP)/openssl/include -ffunction-sections -fdata-sections -Wl,--gc-sections" \
-	CPPFLAGS="$(COPTS) $(MIPS16_OPT) -I$(TOP)/openssl/include -ffunction-sections -fdata-sections -Wl,--gc-sections" \
-	LDFLAGS="$(COPTS) $(MIPS16_OPT) -L$(TOP)/openssl" \
-	$(MAKE) -C libevent
+	LDFLAGS="$(COPTS) $(MIPS16_OPT) -L$(TOP)/openssl" 
\ No newline at end of file
