diff -ruN a/src/Makefile.in b/src/Makefile.in
--- a/src/Makefile.in	2021-02-27 00:11:06.020000000 +0100
+++ b/src/Makefile.in	2021-02-27 00:12:45.490000000 +0100
@@ -846,7 +846,7 @@
 
 errnos-sym.h: Makefile mkstrtable.awk errnos.in
 	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=2 -v nogettext=1 \
-		-v prefix=GPG_ERR_ -v namespace=errnos_ \
+		-v prefix=GPG_ERR_ -v pkg_namespace=errnos_ \
 		$(srcdir)/errnos.in >$@
 
 gpg-error.h: Makefile mkheader.awk \
diff -ruN a/src/mkerrcodes1.awk b/src/mkerrcodes1.awk
--- a/src/mkerrcodes1.awk	2021-02-27 00:11:06.020000000 +0100
+++ b/src/mkerrcodes1.awk	2021-02-27 00:12:45.500000000 +0100
@@ -78,7 +78,7 @@
 }
 
 !header {
-  sub (/\#.+/, "");
+  sub (/#.+/, "");
   sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
 
   if (/^$/)
diff -ruN a/src/mkerrnos.awk b/src/mkerrnos.awk
--- a/src/mkerrnos.awk	2021-02-27 00:11:06.020000000 +0100
+++ b/src/mkerrnos.awk	2021-02-27 00:12:45.500000000 +0100
@@ -80,7 +80,7 @@
 }
 
 !header {
-  sub (/\#.+/, "");
+  sub (/#.+/, "");
   sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
 
   if (/^$/)
diff -ruN a/src/mkstrtable.awk b/src/mkstrtable.awk
--- a/src/mkstrtable.awk	2021-02-27 00:11:06.020000000 +0100
+++ b/src/mkstrtable.awk	2021-02-27 00:20:07.120000000 +0100
@@ -77,7 +77,7 @@
 #
 # The variable prefix can be used to prepend a string to each message.
 #
-# The variable namespace can be used to prepend a string to each
+# The variable pkg_namespace can be used to prepend a string to each
 # variable and macro name.
 
 BEGIN {
@@ -102,7 +102,7 @@
       print "/* The purpose of this complex string table is to produce";
       print "   optimal code with a minimum of relocations.  */";
       print "";
-      print "static const char " namespace "msgstr[] = ";
+      print "static const char " pkg_namespace "msgstr[] = ";
       header = 0;
     }
   else
@@ -110,7 +110,7 @@
 }
 
 !header {
-  sub (/\#.+/, "");
+  sub (/#.+/, "");
   sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
 
   if (/^$/)
@@ -150,14 +150,14 @@
   else
     print "  gettext_noop (\"" prefix last_msgstr "\");";
   print "";
-  print "static const int " namespace "msgidx[] =";
+  print "static const int " pkg_namespace "msgidx[] =";
   print "  {";
   for (i = 0; i < coded_msgs; i++)
     print "    " pos[i] ",";
   print "    " pos[coded_msgs];
   print "  };";
   print "";
-  print "#define " namespace "msgidxof(code) (0 ? -1 \\";
+  print "#define " pkg_namespace "msgidxof(code) (0 ? -1 \\";
 
 # Gather the ranges.
   skip = code[0];
