--- /home/stephan/dd-wrt/src/router/rules/libubox.mk	2021-03-31 23:13:07.940000000 +0200
+++ libubox.mk	2021-03-05 19:27:20.773260400 +0100
@@ -4,7 +4,7 @@
 UBOX_PKG_BUILD_DIR=$(TOP)/libubox
 UBOX_CMAKE_OPTIONS=-DBUILD_LUA=off
 UBOX_STAGING_DIR=$(TOP)/_staging
-UBOX_EXTRA_CFLAGS=-I$(TOP)/_staging/usr/include -I$(TOP)/_staging/usr/include/json $(COPTS) $(MIPS16_OPT) 
+UBOX_EXTRA_CFLAGS=-I$(TOP)/_staging/usr/include -I$(TOP)/_staging/usr/include/json-c $(COPTS) $(MIPS16_OPT) 
 UBOX_EXTRA_LDFLAGS=-L$(TOP)/_staging/usr/lib
 
 MAKE_FLAGS+=VERBOSE=0
