--- /home/stephan/dd-wrt/src/router/rules/libxml2.mk	2021-03-31 23:13:07.940000000 +0200
+++ libxml2.mk	2021-03-05 19:27:20.773260400 +0100
@@ -15,7 +15,7 @@
 libxml2-clean:
 	$(MAKE) -C libxml2 clean
 
-libxml2-configure: zlib
+libxml2-configure:
 	cd libxml2 && ./configure ac_cv_host=$(ARCH)-uclibc-linux --target=$(ARCH)-linux --host=$(ARCH) CC="ccache $(ARCH)-linux-uclibc-gcc" --without-python 
 	CFLAGS="$(COPTS) $(MIPS16_OPT) -DOLD_LIBC_MODE -ffunction-sections -fdata-sections -Wl,--gc-section"
 	CC="ccache $(ARCH)-linux-uclibc-gcc" \
