--- /home/stephan/dd-wrt/src/router/Makefile.northstar	2021-03-31 23:13:07.850000000 +0200
+++ Makefile.northstar	2021-03-06 00:20:38.583260400 +0100
@@ -147,10 +147,10 @@
 obj-install := $(foreach obj,$(obj-y),$(obj)-install)
 obj-distclean := $(foreach obj,$(obj-y) $(obj-n),$(obj)-distclean)
 
-all: install_headers build_date clean_target $(obj-y) $(LINUXDIR)/.config kernel
+all: install_headers build_date clean_target $(LINUXDIR)/.config $(obj-y) kernel 
 
 build_date:
-	echo "#define BUILD_DATE \"$(shell date +%D)\"" > ../../opt/build.h 
+	echo "#define BUILD_DATE \"$(shell date +%D)\"" > opt/build.h 
 	echo "CONFIG_MUSL=y" >> .config 
 
 
@@ -356,13 +356,13 @@
 	mkdir -p $(ARCH)-uclibc/target/etc/kaid
 	mkdir -p $(ARCH)-uclibc/target/etc/langpack
 	cp ./opt/bin/ipkg $(ARCH)-uclibc/target/bin
-	cd  $(ARCH)-uclibc/target/lib
+	mkdir -p $(ARCH)-uclibc/target/lib
 	cp ./opt/etc/preinit $(ARCH)-uclibc/target/etc
 	cp ./opt/etc/postinit $(ARCH)-uclibc/target/etc
 	cp ./opt/etc/ipkg-arm.conf $(ARCH)-uclibc/target/etc/ipkg.conf
 	cp ./opt/etc/config/* $(ARCH)-uclibc/target/etc/config
 	cp ./opt/usr/lib/smb.conf $(ARCH)-uclibc/target/usr/lib
-	ln -sf ../tmp/smbshare $(ARCH)-uclibc/target/www/smb
+####	ln -sf ../tmp/smbshare $(ARCH)-uclibc/target/www/smb   ### wieso?
 	# optimize the crypto library by removing unneeded symbols
 	# Install (and possibly optimize) C library
 #	$(MAKE) -f Makefile.northstar lib.$(ARCH)-install
@@ -497,22 +497,22 @@
 
 	cd $(TARGETDIR) && $(TOP)/misc/rootprep.sh
 	$(TOP)/../../tools/removewhitespace.sh $(TOP)/../../tools $(TARGETDIR)/etc/config
-	$(LINUXDIR)/scripts/squashfs/mksquashfs4 $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -comp xz -nopad  -root-owned -noappend -Xbcj arm -b 262144
-#	$(LINUXDIR)/scripts/squashfs/mksquashfs-lzma $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -noappend -root-owned -le
+	/usr/bin/mksquashfs $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -comp xz -nopad  -root-owned -noappend -Xbcj arm -b 262144
+	$(LINUXDIR)/scripts/squashfs/mksquashfs-lzma $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -noappend -root-owned -le
 	mkfs.jffs2 --pad --big-endian --squash -e 0x20000 -o $(ARCH)-uclibc/dd-wrt.jffs2 -d $(ARCH)-uclibc/target
 	tools/lzma_4k e $(LINUXDIR)/arch/arm/boot/zImage $(ARCH)-uclibc/vmlinuz.lzma
 	echo -ne '\xde\xad\xc0\xde' >> $(ARCH)-uclibc/dd-wrt.jffs2
 #	./tools/trx -m 32000000 -o $(ARCH)-uclibc/northstar-firmware-jffs2.bin $(ARCH)-uclibc/vmlinuz.lzma $(ARCH)-uclibc/dd-wrt.jffs2
 	dd if=/dev/zero of=$(ARCH)-uclibc/pad bs=65536 count=1
 	cat $(ARCH)-uclibc/pad >> $(ARCH)-uclibc/root.fs
-	./tools/trx -m 40000000 -o $(ARCH)-uclibc/northstar-firmware-squashfs.bin $(ARCH)-uclibc/vmlinuz.lzma -a 131072 $(ARCH)-uclibc/root.fs
-	./tools/trx -m 40000000 -o tools/tplink/kernel.hdr0 $(ARCH)-uclibc/vmlinuz.lzma
-	cd tools/tplink && ./mktplinkfw3 -B ARCHERC9 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc9.bin
-	cd tools/tplink && ./mktplinkfw3 -B ARCHERC9v2 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc9v2.bin
-	cd tools/tplink && ./mktplinkfw3 -B ARCHERC1900 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc1900.bin
-	cd tools/tplink && ./mktplinkfw3 -B ARCHERC8 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc8.bin
-	cd tools/tplink && ./mktplinkfw3 -B ARCHERC8v2 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc8_v2.bin
-	cd tools/tplink && ./mktplinkfw3 -B ARCHERC8v3 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc8_v3.bin
+#####	$(TOP)/../../tools/trx -m 40000000 -o $(ARCH)-uclibc/northstar-firmware-squashfs.bin $(ARCH)-uclibc/vmlinuz.lzma -a 131072 $(ARCH)-uclibc/root.fs
+#####	$(TOP)/../../tools/trx -m 40000000 -o tools/tplink/kernel.hdr0 $(ARCH)-uclibc/vmlinuz.lzma
+#####		cd tools/tplink && ./mktplinkfw3 -B ARCHERC9 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc9.bin
+#####		cd tools/tplink && ./mktplinkfw3 -B ARCHERC9v2 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc9v2.bin
+#####		cd tools/tplink && ./mktplinkfw3 -B ARCHERC1900 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc1900.bin
+#####		cd tools/tplink && ./mktplinkfw3 -B ARCHERC8 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc8.bin
+#####		cd tools/tplink && ./mktplinkfw3 -B ARCHERC8v2 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc8_v2.bin
+#####		cd tools/tplink && ./mktplinkfw3 -B ARCHERC8v3 -k kernel.hdr0 -r ../../arm-uclibc/root.fs -o ../../$(ARCH)-uclibc/archerc8_v3.bin
 
 	-./tools/safeloader/tplink-safeloader -B ARCHER-C9-V3 -V 1 -k tools/tplink/kernel.hdr0 -r $(ARCH)-uclibc/root.fs -o $(ARCH)-uclibc/tplink-ARCHER-C9-v3-firmware.bin
 #	-./tools/safeloader/tplink-safeloader -B ARCHER-C9-V3 -V 1 -S -k $(ARCH)-uclibc/kernel.hdr0 -r $(ARCH)-uclibc/root.fs -o $(ARCH)-uclibc/archerC9v3_tp_recovery.bin
@@ -525,7 +525,7 @@
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir868.img -i $(ARCH)-uclibc/raw-dir868.img.seama -m signature=wrgac02_dlob.hans_dir868
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir868.img
 	dd if=$(ARCH)-uclibc/web-dir868.img of=$(ARCH)-uclibc/webflash-dir868.img skip=48 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir868.trx $(ARCH)-uclibc/webflash-dir868.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir868.trx $(ARCH)-uclibc/webflash-dir868.img
 
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir868b.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
@@ -533,14 +533,14 @@
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir868b.img -i $(ARCH)-uclibc/raw-dir868b.img.seama -m signature=wrgac02_dlink.2013gui_dir868lb
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir868b.img
 	dd if=$(ARCH)-uclibc/web-dir868b.img of=$(ARCH)-uclibc/webflash-dir868b.img skip=56 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir868b.trx $(ARCH)-uclibc/webflash-dir868b.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir868b.trx $(ARCH)-uclibc/webflash-dir868b.img
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir868c.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
 	./tools/seama/seama -i $(ARCH)-uclibc/raw-dir868c.img -m dev=/dev/mtdblock/7 -m type=firmware 
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir868c.img -i $(ARCH)-uclibc/raw-dir868c.img.seama -m signature=wrgac35_dlink.2013gui_dir868lc
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir868c.img
 	dd if=$(ARCH)-uclibc/web-dir868c.img of=$(ARCH)-uclibc/webflash-dir868c.img skip=56 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir868c.trx $(ARCH)-uclibc/webflash-dir868c.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir868c.trx $(ARCH)-uclibc/webflash-dir868c.img
 
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir860.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
@@ -548,14 +548,14 @@
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir860.img -i $(ARCH)-uclibc/raw-dir860.img.seama -m signature=wrgac03_dlob.hans_dir860
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir860.img
 	dd if=$(ARCH)-uclibc/web-dir860.img of=$(ARCH)-uclibc/webflash-dir860.img skip=48 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir860.trx $(ARCH)-uclibc/webflash-dir860.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir860.trx $(ARCH)-uclibc/webflash-dir860.img
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir890.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
 	./tools/seama/seama -i $(ARCH)-uclibc/raw-dir890.img -m dev=/dev/mtdblock/7 -m type=firmware 
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir890.img -i $(ARCH)-uclibc/raw-dir890.img.seama -m signature=wrgac36_dlink.2013gui_dir890
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir890.img
 	dd if=$(ARCH)-uclibc/web-dir890.img of=$(ARCH)-uclibc/webflash-dir890.img skip=52 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir890.trx $(ARCH)-uclibc/webflash-dir890.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir890.trx $(ARCH)-uclibc/webflash-dir890.img
 
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir885.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
@@ -563,14 +563,14 @@
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir885.img -i $(ARCH)-uclibc/raw-dir885.img.seama -m signature=wrgac42_dlink.2015_dir885l
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir885.img
 	dd if=$(ARCH)-uclibc/web-dir885.img of=$(ARCH)-uclibc/webflash-dir885.img skip=52 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir885.trx $(ARCH)-uclibc/webflash-dir885.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir885.trx $(ARCH)-uclibc/webflash-dir885.img
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir895.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
 	./tools/seama/seama -i $(ARCH)-uclibc/raw-dir895.img -m dev=/dev/mtdblock/7 -m type=firmware 
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir895.img -i $(ARCH)-uclibc/raw-dir895.img.seama -m signature=wrgac40_dlink.2015_dir895l
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir895.img
 	dd if=$(ARCH)-uclibc/web-dir895.img of=$(ARCH)-uclibc/webflash-dir895.img skip=52 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir895.trx $(ARCH)-uclibc/webflash-dir895.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir895.trx $(ARCH)-uclibc/webflash-dir895.img
 
 
 	./tools/seama/packimgs -b 64 -o $(ARCH)-uclibc/raw-dir880.img -i $(ARCH)-uclibc/vmlinuz.lzma -i $(ARCH)-uclibc/root.fs
@@ -578,24 +578,24 @@
 	./tools/seama/seama -s $(ARCH)-uclibc/web-dir880.img -i $(ARCH)-uclibc/raw-dir880.img.seama -m signature=wrgac16_dlink.2013gui_dir880
 	./tools/seama/seama -d $(ARCH)-uclibc/web-dir880.img
 	dd if=$(ARCH)-uclibc/web-dir880.img of=$(ARCH)-uclibc/webflash-dir880.img skip=52 iflag=skip_bytes 
-	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir880.trx $(ARCH)-uclibc/webflash-dir880.img
+####	./tools/trx_n -m 40000000 -o $(ARCH)-uclibc/webflash-dir880.trx $(ARCH)-uclibc/webflash-dir880.img
 
 	
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC56U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac56u-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC56U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac56u-firmware.trx
 #	./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC67U,3.0.0.4,$(ARCH)-uclibc/asus_rt-ac67u-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC68U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac68u-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC87U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac87u-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC88U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac88u-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC5300,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac5300-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC3200,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac3200-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC3100,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac3100-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-N18U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-n18u-firmware.trx
-	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC1200G+,3.0.0.4,380,1234,$(ARCH)-uclibc/asus_rt-ac1200gp-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC68U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac68u-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC87U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac87u-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC88U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac88u-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC5300,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac5300-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC3200,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac3200-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC3100,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-ac3100-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-N18U,3.0.0.4,380,760,$(ARCH)-uclibc/asus_rt-n18u-firmware.trx
+####	-./tools/trx_asus -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC1200G+,3.0.0.4,380,1234,$(ARCH)-uclibc/asus_rt-ac1200gp-firmware.trx
 #	-./tools/trx_asus2 -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC1200G+,3.0.0.4,380,1234,$(ARCH)-uclibc/asus_rt-ac1200gp-firmware.trx2
 #	-./tools/trx_asus3 -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC1200G+,3.0.0.4,380,1234,$(ARCH)-uclibc/asus_rt-ac1200gp-firmware.trx3
 
 	dd if=$(ARCH)-uclibc/northstar-firmware-squashfs.bin of=$(ARCH)-uclibc/buffalo.trx bs=131072 conv=sync
-	cd $(ARCH)-uclibc && ../tools/bufenc/makefw.sh buffalo
+####	cd $(ARCH)-uclibc && ../tools/bufenc/makefw.sh buffalo
 
 	cp $(ARCH)-uclibc/northstar-firmware-squashfs.bin $(ARCH)-uclibc/netgear.trx
 ifeq ($(CONFIG_DHDAP),y)
