--- Makefile.northstar.orig	2022-08-07 15:16:19.018337000 +0200
+++ Makefile.northstar	2022-08-15 18:35:57.865177125 +0200
@@ -225,14 +225,15 @@
 	-$(MAKE) -f Makefile.northstar batman-adv-install
 endif
 ifeq ($(CONFIG_OPENDPI),y)
-	-$(MAKE) -f Makefile.northstar ndpi-netfilter
-	-$(MAKE) -f Makefile.northstar ndpi-netfilter-install
+	-$(MAKE) -C ndpi-netfilter
+	-$(MAKE) -C ndpi-netfilter install
 endif
 ifeq ($(CONFIG_ZFS),y)
 	-$(MAKE) -f Makefile.northstar zfs
 	-$(MAKE) -f Makefile.northstar zfs-install
 endif
 ifeq ($(CONFIG_SMBD),y)
+	-$(MAKE) -f Makefile.northstar smbd-configure
 	-$(MAKE) -f Makefile.northstar smbd
 	-$(MAKE) -f Makefile.northstar smbd-install
 endif
@@ -241,10 +242,10 @@
 	-$(MAKE) -f Makefile.northstar wireguard-install
 endif
 ifeq ($(CONFIG_CAKE),y)
-	-$(MAKE) -f Makefile.northstar cake
-	-$(MAKE) -f Makefile.northstar cake-install
-	-$(MAKE) -f Makefile.northstar fq_codel_fast
-	-$(MAKE) -f Makefile.northstar fq_codel_fast-install
+	-$(MAKE) -C cake
+	-$(MAKE) -C cake install
+	-$(MAKE) -C fq_codel_fast
+	-$(MAKE) -C fq_codel_fast install
 endif
 
 	rm -f $(TARGETDIR)/lib/modules/$(KERNELRELEASE)/build
@@ -346,7 +347,7 @@
 		rm -f $(EX_PATH) ; \
 	fi
 
-install package: clean_target $(filter-out lib.$(ARCH)-install,$(obj-install)) $(LINUXDIR)/.config
+install package: clean_target $(filter-out lib.mipsel-install,$(obj-install))
         # Install binaries into target directory
 	install -d $(TARGETDIR)
 	for dir in $(wildcard $(patsubst %,$(INSTALLDIR)/%,$(obj-y))) ; do \
@@ -497,7 +498,7 @@
 
 	cd $(TARGETDIR) && $(TOP)/misc/rootprep.sh
 	$(TOP)/../../tools/removewhitespace.sh $(TOP)/../../tools $(TARGETDIR)/etc/config
-	$(LINUXDIR)/scripts/squashfs/mksquashfs4 $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -comp xz -nopad  -root-owned -noappend -Xbcj arm -b 262144
+	mksquashfs $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -comp xz -nopad  -root-owned -noappend -Xbcj arm -b 262144
 #	$(LINUXDIR)/scripts/squashfs/mksquashfs-lzma $(ARCH)-uclibc/target $(ARCH)-uclibc/root.fs -noappend -root-owned -le
 	mkfs.jffs2 --pad --big-endian --squash -e 0x20000 -o $(ARCH)-uclibc/dd-wrt.jffs2 -d $(ARCH)-uclibc/target
 	tools/lzma_4k e $(LINUXDIR)/arch/arm/boot/zImage $(ARCH)-uclibc/vmlinuz.lzma
@@ -595,7 +596,7 @@
 #	-./tools/trx_asus3 -i $(ARCH)-uclibc/northstar-firmware-squashfs.bin -r RT-AC1200G+,3.0.0.4,380,1234,$(ARCH)-uclibc/asus_rt-ac1200gp-firmware.trx3
 
 	dd if=$(ARCH)-uclibc/northstar-firmware-squashfs.bin of=$(ARCH)-uclibc/buffalo.trx bs=131072 conv=sync
-	cd $(ARCH)-uclibc && ../tools/bufenc/makefw.sh buffalo
+	-cd $(ARCH)-uclibc && ../tools/bufenc/makefw.sh buffalo
 
 	cp $(ARCH)-uclibc/northstar-firmware-squashfs.bin $(ARCH)-uclibc/netgear.trx
 ifeq ($(CONFIG_DHDAP),y)
@@ -638,6 +639,7 @@
 
 include rules/all.mk
 
+
 .PHONY: all clean distclean mrproper install package
 .PHONY: conf mconf oldconf kconf kmconf config menuconfig oldconfig
 .PHONY: dummy fixkc libnet libpcap
