--- Makefile.orig	2022-08-11 12:57:43.127036193 +0200
+++ Makefile	2022-08-11 12:57:48.757273535 +0200
@@ -105,12 +105,12 @@
 	cp hotspotsystem/hotspotsystem.png $(INSTALLDIR)/www/images
 endif
 
-ifeq ($(CONFIG_3COM),y)
-	
-	mkdir $(INSTALLDIR)/www/style/3com
-	cp 3com/style.css $(INSTALLDIR)/www/style/3com
-	cp 3com/images/* $(INSTALLDIR)/www/images
-endif
+#ifeq ($(CONFIG_3COM),y)
+#	
+#	mkdir $(INSTALLDIR)/www/style/3com
+#	cp 3com/style.css $(INSTALLDIR)/www/style/3com
+#	cp 3com/images/* $(INSTALLDIR)/www/images
+#endif
 
 ifeq ($(CONFIG_BUFFALO),y)
 ifneq ($(CONFIG_SPOTPASS),y)
