--- /home/stephan/dd-wrt/src/router/kromo/dd-wrt/Makefile	2021-03-31 23:13:07.900000000 +0200
+++ Makefile_kromo	2021-03-14 14:27:10.250000000 +0100
@@ -102,12 +102,12 @@
 	cp hotspotsystem/hotspotsystem.png $(INSTALLDIR)/www/images
 endif
 
-ifeq ($(CONFIG_3COM),y)
-	
-	mkdir $(INSTALLDIR)/www/style/3com
-	cp 3com/style.css $(INSTALLDIR)/www/style/3com
-	cp 3com/images/* $(INSTALLDIR)/www/images
-endif
+#ifeq ($(CONFIG_3COM),y)
+#	
+#	mkdir $(INSTALLDIR)/www/style/3com
+#	cp 3com/style.css $(INSTALLDIR)/www/style/3com
+#	cp 3com/images/* $(INSTALLDIR)/www/images
+#endif
 
 ifeq ($(CONFIG_BUFFALO),y)
 ifneq ($(CONFIG_SPOTPASS),y)
