--- /home/stephan/dd-wrt/src/router/minidlna/minidlna/Makefile	2021-03-31 23:13:07.900000000 +0200
+++ Makefile_minidlna_minidlna	2021-03-05 19:27:20.773260400 +0100
@@ -13,7 +13,10 @@
 #CFLAGS = -Wall -O -D_GNU_SOURCE -g -DDEBUG
 #CFLAGS = -Wall -g -Os -D_GNU_SOURCE
 
-LDFLAGS+= -Wl,--gc-sections
+LDFLAGS+= -Wl,--gc-sections -L$(TOP)/zlib -L$(MINI_DLNA_PATH)/sqlite-3.6.22/.libs -L$(MINI_DLNA_PATH)/jpeg-8 -L$(MINI_DLNA_PATH)/libvorbis-1.3.3/lib/.libs \
+		-L$(MINI_DLNA_PATH)/libogg-1.3.0/src/.libs -L$(MINI_DLNA_PATH)/libexif-0.6.19/libexif/.libs -L$(MINI_DLNA_PATH)/flac-1.2.1/src/libFLAC/.libs \
+		-L$(MINI_DLNA_PATH)/ffmpeg-3.1/libavutil -L$(MINI_DLNA_PATH)/ffmpeg-3.1/libavcodec -L$(MINI_DLNA_PATH)/ffmpeg-3.1/libavformat \
+		-L$(MINI_DLNA_PATH)/libid3tag-0.15.0b/.libs
 CFLAGS = -Wall $(COPTS) $(MIPS16_OPT) $(LTO) -ffunction-sections -fdata-sections -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -DHAVE_LIBAVUTIL_AVUTIL_H -DHAVE_LIBAVCODEC_AVCODEC_H -DHAVE_LIBAVFORMAT_AVFORMAT_H -DHAVE_VORBISFILE -DNEED_OGG\
          -I$(MINI_DLNA_PATH)/ffmpeg-3.1 \
          -I$(MINI_DLNA_PATH)/ffmpeg-3.1/libavcodec \
