--- /home/stephan/dd-wrt/src/router/transmission/daemon/Makefile.am	2021-03-31 23:13:07.980000000 +0200
+++ Makefile_transmission_daemon.am	2021-03-21 13:57:15.580000000 +0100
@@ -3,7 +3,6 @@
 AM_CFLAGS = \
     @LIBEVENT_CFLAGS@ \
     @LIBCURL_CFLAGS@ \
-    @SYSTEMD_CFLAGS@ \
     @ZLIB_CFLAGS@ \
     @PTHREAD_CFLAGS@
 
@@ -27,7 +26,6 @@
     @LIBCURL_LIBS@ \
     @CRYPTO_LIBS@ \
     @INTLLIBS@ \
-    @SYSTEMD_LIBS@ \
     @ZLIB_LIBS@ \
     @PTHREAD_LIBS@ \
     ${LIBM}
