--- mtr.mk.orig	2022-08-07 15:16:38.628785000 +0200
+++ mtr.mk	2022-07-26 16:19:05.240586000 +0200
@@ -1,6 +1,6 @@
 mtr-configure: ncurses-configure ncurses
 	cd mtr && autoreconf -fi
-	cd mtr && ./configure --disable-nls --without-glib --without-gtk --prefix=/usr --host=$(ARCH)-linux CC="$(CC)" CFLAGS="$(COPTS) $(MIPS16_OPT) -D__GLIBC__ -I$(TOP)/ncurses/include -L$(TOP)/ncurses/lib  -ffunction-sections -fdata-sections -Wl,--gc-sections -DNEED_PRINTF  -lncurses" --without-gtk ac_cv_lib_resolv_res_mkquery=yes
+	cd mtr && ./configure --disable-nls --without-glib --without-gtk --prefix=/usr --host=$(ARCH)-linux CC="$(CC)" CFLAGS="$(COPTS) $(MIPS16_OPT) -D__GLIBC__ -I$(TOP)/ncurses/include -L$(TOP)/ncurses/lib  -I$(TOP)/jansson/src -L$(TOP)/jansson/src/.libs -ffunction-sections -fdata-sections -Wl,--gc-sections -DNEED_PRINTF  -lncurses" --without-gtk ac_cv_lib_resolv_res_mkquery=yes
 	cd mtr && touch *
 
 
