--- nettle.mk.orig	2022-08-07 15:16:38.628785000 +0200
+++ nettle.mk	2022-07-29 19:00:52.269312000 +0200
@@ -6,7 +6,7 @@
 NETTLE_CONFIGURE_ARGS+= --disable-assembler
 endif
 nettle-configure: pcre zlib openssl gmp
-	cd nettle && ./configure --host=$(ARCH)-linux --disable-shared --enable-static --disable-pic --prefix=/usr --libdir=/usr/lib $(NETTLE_CONFIGURE_ARGS) CFLAGS="$(COPTS) $(LTO) $(MIPS16_OPT) -fPIC -ffunction-sections -fdata-sections -I$(TOP)/pcre -I$(TOP)/gmp -I$(TOP)/zlib  -fPIC" CPPFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC" LDFLAGS="-L$(TOP)/pcre/.libs -L$(TOP)/gmp/.libs -lpthread -lpcre -L$(TOP)/zlib $(LDFLAGS) $(LDLTO) -lz" \
+	cd nettle && autoreconf -fi && ./configure --host=$(ARCH)-linux --disable-shared --enable-static --disable-pic --prefix=/usr --libdir=/usr/lib $(NETTLE_CONFIGURE_ARGS) CFLAGS="$(COPTS) $(LTO) $(MIPS16_OPT) -fPIC -ffunction-sections -fdata-sections -I$(TOP)/pcre -I$(TOP)/gmp -I$(TOP)/zlib  -fPIC" CPPFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC" LDFLAGS="-L$(TOP)/pcre/.libs -L$(TOP)/gmp/.libs -lpthread -lpcre -L$(TOP)/zlib $(LDFLAGS) $(LDLTO) -lz" \
 	AR_FLAGS="cru $(LTOPLUGIN)" \
 	RANLIB="$(ARCH)-linux-ranlib $(LTOPLUGIN)"
 
