--- ntfs-3g.mk.orig	2022-08-11 12:57:43.133703143 +0200
+++ ntfs-3g.mk	2022-08-11 12:57:48.697271019 +0200
@@ -1,11 +1,11 @@
 
-ntfs-3g:
+ntfs-3g: ntfs-3g-configure
 	CC="$(CC)" \
-	CFLAGS="$(COPTS) -DNEED_PRINTF $(MIPS16_OPT) -ffunction-sections -fdata-sections -Wl,--gc-sections" \
+	CFLAGS="$(COPTS)  $(MIPS16_OPT) -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	CPPFLAGS="$(COPTS) $(MIPS16_OPT) -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	CXXFLAGS="$(COPTS) $(MIPS16_OPT) -ffunction-sections -fdata-sections -Wl,--gc-sections" \
 	LDFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC -ffunction-sections -fdata-sections -Wl,--gc-sections" \
-	$(MAKE) -C ntfs-3g/fuse
+#	$(MAKE) -C ntfs-3g/fuse
 	$(MAKE) -C ntfs-3g all
 
 ntfs-3g-install:
