--- /home/stephan/dd-wrt/src/router/rules/pcre.mk	2021-03-31 23:13:07.950000000 +0200
+++ pcre.mk	2021-03-05 19:27:20.783260400 +0100
@@ -1,9 +1,9 @@
 pcre-configure:
-	cd pcre && ./configure --host=$(ARCH)-linux-uclibc CFLAGS="$(COPTS)  $(MIPS16_OPT)" --prefix=/usr --disable-xmldoc --enable-utf8 --enable-unicode-properties --disable-pcretest-libreadline --libdir=$(TOP)/pcre/.libs
+	cd pcre && autoreconf -fi && ./configure --host=$(ARCH)-linux-uclibc CFLAGS="$(COPTS)  $(MIPS16_OPT)" --prefix=/usr --disable-xmldoc --enable-utf8 --enable-unicode-properties --disable-pcretest-libreadline --libdir=$(TOP)/pcre/.libs
 	touch $(TOP)/pcre/*   
 
 
-pcre:
+pcre: pcre-configure
 	$(MAKE) -C pcre CFLAGS="$(COPTS) $(MIPS16_OPT)" CXXFLAGS="$(COPTS) $(MIPS16_OPT)" CPPFLAGS="$(COPTS) $(MIPS16_OPT)"
 
 pcre-clean:
