--- /home/stephan/dd-wrt/src/router/rules/pptpd.mk	2021-03-31 23:13:07.950000000 +0200
+++ pptpd.mk	2021-03-14 16:11:15.950000000 +0100
@@ -12,7 +12,7 @@
 	$(MAKE) -C pptpd-accel
 else
 	CFLAGS="$(COPTS) $(MIPS16_OPT) -D_GNU_SOURCE -ffunction-sections -fdata-sections -Wl,--gc-sections   -I$(TOP)/pppd/include -DNEED_PRINTF" \
-	CPPFLAGS="$(COPTS)$(MIPS16_OPT) -D_GNU_SOURCE -ffunction-sections -fdata-sections -Wl,--gc-sections   -I$(TOP)/pppd/include" \
+	CPPFLAGS="$(COPTS) $(MIPS16_OPT) -D_GNU_SOURCE -ffunction-sections -fdata-sections -Wl,--gc-sections   -I$(TOP)/pppd/include" \
 	$(MAKE) -C pptpd
 endif
 else
