--- /home/stephan/dd-wrt/src/router/rules/quagga.mk	2021-03-31 23:13:07.950000000 +0200
+++ quagga.mk	2021-03-05 19:27:20.783260400 +0100
@@ -1,4 +1,4 @@
-quagga-configure: ncurses readline
+quagga-configure: 
 	cd quagga && autoreconf --force --install
 	cd quagga && ./configure --host=$(ARCH)-uclibc-linux $(CONFIG_QUAGGA_EXTRA) --localstatedir=/var/run  --libdir=/usr/tmp --enable-opaque-lsa --disable-nhrpd --enable-ospf-te --disable-ospfclient --enable-multipath=32  --enable-ipv6 --prefix=/usr --sysconfdir=/tmp --disable-ospf6d  --enable-vtysh --enable-user=root --enable-group=root --disable-ospfapi --disable-isisd --disable-pimd --disable-nhrpd --enable-pie=no --with-libreadline=$(TOP)/readline CFLAGS="-fno-strict-aliasing -I$(TOP)/ -DNEED_PRINTF -fcommon -Drpl_malloc=malloc -Drpl_realloc=realloc $(COPTS)  $(MIPS16_OPT)" LDFLAGS="-L$(TOP)/readline/shlib -L$(TOP)/ncurses/lib -lncurses" 
 	cd quagga && touch *
