--- /home/stephan/dd-wrt/src/router/rules/readline.mk	2021-03-31 23:13:07.950000000 +0200
+++ readline.mk	2021-03-25 19:17:48.760000000 +0100
@@ -1,10 +1,10 @@
 readline-configure: 
 	cd readline && ./configure --host=$(ARCH)-uclibc-linux --prefix=/usr --libdir=/usr/lib --disable-static CFLAGS="$(COPTS) $(MIPS16_OPT) -fPIC -fno-strict-aliasing"
-	$(MAKE) -C readline clean all
+	$(MAKE) -C readline clean
 	cd readline/shlib && rm -f libreadline.so
 	cd readline/shlib && rm -f libhistory.so
 
-readline:
+readline: readline-configure
 	$(MAKE) -C readline
 	cd readline/shlib && rm -f libreadline.so
 	cd readline/shlib && rm -f libhistory.so
