--- /home/stephan/dd-wrt/src/router/samba36/source3/autogen.sh	2021-03-31 23:13:07.950000000 +0200
+++ samba36_patches/autogen.sh	2021-03-17 23:39:52.250000000 +0100
@@ -87,9 +87,9 @@
 if gcc -E tests/preproc-dummy.c -o /dev/null ;
 then
     PIDL_OUTPUTDIR="librpc/gen_ndr" CPP="gcc -E" PIDL=../pidl/pidl \
-	srcdir=. ../librpc/build_idl.sh ../librpc/idl/*.idl
+	srcdir=. ../librpc/build_idl.sh --full ../librpc/idl/*.idl
     PIDL_OUTPUTDIR="librpc/gen_ndr" CPP="gcc -E" PIDL=../pidl/pidl \
-	srcdir=. script/build_idl.sh librpc/idl/*.idl
+	srcdir=. script/build_idl.sh --full librpc/idl/*.idl
 else
    echo "Warning: Could not compile idl files in autogen, "
    echo "some autconf tests might not work properly"
