--- /home/stephan/dd-wrt/src/router/rules/samba3.mk	2021-03-31 23:13:07.950000000 +0200
+++ samba3.mk	2021-03-18 00:00:13.480000000 +0100
@@ -16,7 +16,8 @@
 	samba_cv_USE_SETREUID=yes \
 	samba_cv_USE_SETRESUID=yes \
 	samba_cv_have_setreuid=yes \
-	samba_cv_have_setresuid=yes
+	samba_cv_have_setresuid=yes \
+	libreplace_cv_HAVE_GETADDRINFO=no
 
 CONFIGURE_ARGS_SMB += \
 	--host=$(ARCH)-linux \
@@ -51,14 +52,17 @@
 	--without-libsmbclient \
 	--without-libsmbsharemodes \
 	--without-libaddns \
+	--disable-external-libtalloc \
+	--disable-external-libtdb \
+	--disable-external-libtevent \
 	--with-shared-modules=pdb_tdbsam,pdb_wbc_sam,idmap_nss,nss_info_template,auth_winbind,auth_wbc,auth_domain
 
 samba3-preconfigure:
-	if ! test -e "samba36/source3/Makefile"; then	cd samba36/source3 && ./configure $(CONFIGURE_VARS) $(CONFIGURE_ARGS_SMB) CFLAGS="$(COPTS) $(LTO) -DMAX_DEBUG_LEVEL=-1  -ffunction-sections -fdata-sections -Wl,--gc-sections $(SAMBA3_EXTRA)" LDFLAGS="$(COPTS) $(LDLTO) -DMAX_DEBUG_LEVEL=-1  -ffunction-sections -fdata-sections -Wl,--gc-sections $(SAMBA3_EXTRA)"; fi
+#	if ! test -e "samba36/source3/Makefile"; then	
+	cd samba36/source3 && ./autogen.sh && ./configure $(CONFIGURE_VARS) $(CONFIGURE_ARGS_SMB) CFLAGS="$(COPTS) $(LTO) -DMAX_DEBUG_LEVEL=-1  -ffunction-sections -fdata-sections -Wl,--gc-sections -std=gnu99 $(SAMBA3_EXTRA)" LDFLAGS="$(COPTS) $(LDLTO) -DMAX_DEBUG_LEVEL=-1  -ffunction-sections -fdata-sections -Wl,--gc-sections $(SAMBA3_EXTRA)"
 
 samba3-configure: samba3-delete samba3-preconfigure
 	touch samba36/librpc/idl/*
-	
 
 samba3-delete:
 	rm -f samba36/source3/Makefile
@@ -66,16 +70,13 @@
 samba3: samba3-preconfigure
 	install -D samba36/config/samba3.webnas httpd/ej_temp/02samba3.webnas
 	install -D filesharing/config/zfilesharing.webnas httpd/ej_temp/03zfilesharing.webnas
-	$(MAKE) -C samba36/source3 bin/libtalloc.a WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
-	$(MAKE) -C samba36/source3 bin/libtdb.a WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
-	$(MAKE) -C samba36/source3 bin/libtevent.a WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
-	$(MAKE) -C samba36/source3 bin/libwbclient.a WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
+	$(MAKE) -C samba36/source3 libtalloc WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
+	$(MAKE) -C samba36/source3 libtdb WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
+	$(MAKE) -C samba36/source3 libtevent WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
+	$(MAKE) -C samba36/source3 libwbclient WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
 	$(MAKE) -C samba36/source3 bin/samba_multicall WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
 	$(MAKE) -C samba36/source3 bin/smbpasswd WITH_LFS=yes DYNEXP= PICFLAG= MODULES= 
 
-
-
-
 samba3-install:
 	mkdir -p $(INSTALLDIR)/samba3
 	install -D samba36/source3/bin/samba_multicall $(INSTALLDIR)/samba3/usr/sbin/samba_multicall
