--- tor.mk.orig	2021-04-03 14:09:30.840000000 +0200
+++ tor.mk	2021-04-03 14:16:09.760000000 +0200
@@ -5,7 +5,7 @@
 	make -C miniupnpc clean
 
 miniupnpc-configure:
-	make -C miniupnpc
+	
 
 miniupnpc-install:
 	make -C miniupnpc install INSTALLPREFIX=$(INSTALLDIR)/miniupnpc/usr 
@@ -14,7 +14,7 @@
 	rm -rf $(INSTALLDIR)/miniupnpc/usr/share
 	rm -f $(INSTALLDIR)/miniupnpc/usr/lib/*.a
 
-tor-configure: openssl xz zstd libevent zlib
+tor-configure:
 	cd tor && libtoolize -ci --force 
 	cd tor && aclocal
 	cd tor && automake --add-missing
@@ -35,7 +35,7 @@
 	ZSTD_CFLAGS="-I$(TOP)/zstd/lib" \
 	ZSTD_LIBS="-I$(TOP)/zstd/lib -lzstd"
 
-tor: openssl libevent zlib
+tor:
 	install -D tor/config/tor.webservices httpd/ej_temp/
 	make -C tor ARFLAGS="cru $(LTOPLUGIN)"  all
 
