--- /home/stephan/dd-wrt/src/router/rules/transmission.mk	2021-03-31 23:13:07.950000000 +0200
+++ transmission.mk	2021-03-22 09:35:40.500000000 +0100
@@ -1,15 +1,5 @@
-transmission: libevent curl zlib
+transmission:
 	install -D transmission/configs/transmission.webnas httpd/ej_temp/06transmission.webnas
-	CC="ccache $(ARCH)-linux-uclibc-gcc" \
-	CFLAGS="$(COPTS) -D_GNU_SOURCE -ffunction-sections -fdata-sections -Wl,--gc-sections  -I$(TOP)/zlib  -I$(TOP)/curl/include -I$(TOP)/libevent/include  $(LTO)" \
-	CPPFLAGS="$(COPTS) -D_GNU_SOURCE -ffunction-sections -fdata-sections -Wl,--gc-sections  -I$(TOP)/zlib   -I$(TOP)/curl/include -I$(TOP)/libevent/include  $(LTO)" \
-	LDFLAGS="$(COPTS) -D_GNU_SOURCE -L$(TOP)/zlib -L$(TOP)/libevent/.libs -levent -L$(TOP)/curl/lib/.libs -lcurl -ldl  $(LDLTO)" \
-	ZLIB_CFLAGS="-I$(TOP)/zlib" \
-	ZLIB_LIBS="-L$(TOP)/zlib -lz" \
-	OPENSSL_CFLAGS="-I$(TOP)/openssl/include" \
-	OPENSSL_LIBS="-L$(TOP)/openssl -lssl -lcrypto" \
-	AR_FLAGS="cru $(LTOPLUGIN)" \
-	RANLIB="$(ARCH)-linux-ranlib $(LTOPLUGIN)"
 	$(MAKE) -C transmission
 
 transmission-install:
@@ -31,10 +21,10 @@
 transmission-clean:
 	$(MAKE) -C transmission clean
 
-transmission-configure: libevent-configure curl-configure zlib
-	-cd transmission && ./autogen.sh
+transmission-configure: 
+	cd transmission && ./autogen.sh
 	cd transmission && ./configure  --prefix=/usr ac_cv_host=$(ARCH)-uclibc-linux --target=$(ARCH)-linux --host=$(ARCH) CC="ccache $(ARCH)-linux-uclibc-gcc" \
-	--enable-daemon --enable-lightweight --disable-nls --without-systemd-daemon --libdir=/usr/lib --disable-static --disable-dependency-tracking \
+	--enable-daemon --enable-lightweight --disable-nls --libdir=/usr/lib --disable-static --disable-dependency-tracking --without-gtk \
 	CFLAGS="$(COPTS) -D_GNU_SOURCE -DNO_SYS_QUEUE_H -ffunction-sections -fdata-sections -Wl,--gc-sections  -I$(TOP)/zlib   -I$(TOP)/curl/include -I$(TOP)/openssl/include -I$(TOP)/libevent/include $(LTO)" \
 	CPPFLAGS="$(COPTS) -D_GNU_SOURCE -DNO_SYS_QUEUE_H -ffunction-sections -fdata-sections -Wl,--gc-sections  -I$(TOP)/zlib  -I$(TOP)/curl/include  -I$(TOP)/openssl/include  -I$(TOP)/libevent/include  $(LTO)" \
 	LDFLAGS="$(COPTS) -D_GNU_SOURCE -DNO_SYS_QUEUE_H  -L$(TOP)/zlib   -L$(TOP)/openssl -lssl -lcrypto -L$(TOP)/libevent/.libs  -L$(TOP)/curl/lib/.libs -ldl $(LDLTO)" \
@@ -48,5 +38,3 @@
 	OPENSSL_LIBS="-L$(TOP)/openssl -lssl -lcrypto" \
 	AR_FLAGS="cru $(LTOPLUGIN)" \
 	RANLIB="$(ARCH)-linux-ranlib $(LTOPLUGIN)"
-
-	$(MAKE) -C transmission clean
